.env.example

```
@@ -141,6 +141,20 @@ ENABLE_CUSTOM_PYTHON_FUNCTIONS=false
+# (Twilio X-Twilio-Signature, Plivo V3 signature, Exotel auth_token). Secure by
+# default. Requires TWILIO_AUTH_TOKEN / PLIVO_AUTH_TOKEN / EXOTEL_WEBHOOK_AUTH_TOKEN
+# and a correct APP_BASE_URL for the providers in use. Set to false ONLY as a
+# temporary operational escape hatch — never in production.
+ENFORCE_TELEPHONY_WEBHOOK_SIGNATURES=true
+
+# Public path prefix an ingress strips before requests reach this app. Providers
+# sign the externally-visible URL, so if https://host/agent/voice/... is proxied
+# to /... internally, set this to "/agent/voice" or every legitimate webhook is
+# rejected. Leave empty when no path rewriting happens. Not read from
+# X-Forwarded-Prefix on purpose — that header is caller-controlled.
+TELEPHONY_WEBHOOK_PATH_PREFIX=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)